src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_InterpolatedString.vb

```diff
@@ -184,7 +184,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
 ReturnBadExpression:
             ReportDiagnostic(node, ErrorFactory.ErrorInfo(ERRID.ERR_InterpolatedStringFactoryError, factoryType.Name, factoryMethodName), _diagnostics)
-            Return factory.Convert(targetType, factory.BadExpression(MyBase.VisitInterpolatedStringExpression(node)))
+            Return factory.Convert(targetType, factory.BadExpression(DirectCast(MyBase.VisitInterpolatedStringExpression(node), BoundInterpolatedStringExpression)))
```

Comment:
Based on the signature, VisitInterpolatedStringExpression returns BoundNode. We have an implicit contract that visiting an expression should give at least a BoundExpression back. Is there a reason to make this contract stronger for an Interpolated String Expression?